docs/ref-manual/cli/options.md

```diff
@@ -61,21 +61,28 @@ Adding a message makes it easier to track several runs. It is very useful if you
 To create the message above, we used  
 `certoraRun Bank.sol --verify Bank:Bank.spec --msg 'Removed an assertion'`
 
-### `--rule`
+### `--rule (alternatively: --rules)`
 
 **What does it do?**  
-Formally verifies a single property instead of the whole specification file. An invariant can also be selected.  
+Formally verifies one or more given properties instead of the whole specification file. An invariant can also be selected.  
 **When to use it?**  
-This option saves a lot of run time. Use it whenever you care about only a single property. The most common case is when you add a new rule to an existing specification. The other is when code changes cause a specific rule to fail; in the process of fixing the code, updating the rule, and understanding counterexamples, you likely want to verify only that specific rule.  
+This option saves a lot of run time. Use it whenever you care about only a specific subset of a specification's total properties. The most common case is when you add a new rule to an existing specification. The other is when code changes cause a specific rule to fail; in the process of fixing the code, updating the rule, and understanding counterexamples, you likely want to verify only that specific rule.  
 **Example**  
 If `Bank.spec` includes the following properties:  
 `invariant address_zero_cannot_become_an_account()`
```

Comment:
this invariant is not used in any of the examples below

Reply:
I did not add this line in this PR, it existed before my changes. We can remove/modify this, but that is probably better to do in another PR.